custom-tests.yaml

```diff
@@ -1233,9 +1231,18 @@ api:
     isTypeSupported: return 'isTypeSupported' in MediaSource;
   MediaStream:
     __base: |-
-      <%api.MediaDevices:mediaDevices%>
-      var promise = mediaDevices.getUserMedia({audio: true});
-      promise.then(function() {});
+      // This string makes this a callback test: callback(
```

Comment:
@queengooborg this isn't very nice. Do you have any thoughts on how to write the detection instead? Can we more explicitly mark tests as callback tests?

Reply:
Ping @queengooborg

Reply:
Actually, this string isn't needed!  As long as the string `callback` is present, then we're good to go, no open parenthesis needed!

Reply:
Isn't this the code that determines if it's a callback test?

https://github.com/foolip/mdn-bcd-collector/blob/52df4d5551217feb9f0b1d42b18228caa25a83ef/build.js#L90

There's an open paren there...

Reply:
Oh I was looking at the functions above (we should probably split it off into its own function), hmm...  We could perhaps turn it into a regex check to find a comma or right parenthesis afterwards as well?

Reply:
Can we add the test type as a bit of info in the YAML perhaps, to serve at least as an override for the detected type?

Reply:
So with https://github.com/foolip/mdn-bcd-collector/pull/2253 I can drop the comment here.